lxfspacy.py

```diff
@@ -0,0 +1,37 @@
+import spacy
+
+language_dict = {
+  "pt": {
+    "speed": "pt_core_news_sm",
+    "accuracy": "pt_core_news_lg"
+  },
+  "en": {
+   "speed": "en_core_web_sm",
+   "accuracy": "en_core_web_trf"
+  },
+  "ru": {
+    "speed": "ru_core_news_sm",
+    "accuracy": "ru_core_news_lg"
+  }
+}
+ 
+def nlp(lang, opt):
```

Comment:
This function should be called "load". NLP is what the function returns, not what it does
`def load(lang, opt)`

Reply:
```suggestion
def load(lang, opt):
```